resources/lang/en/settings.php

```diff
@@ -324,6 +326,135 @@
             'description' => 'Device performance entries older than (days)',
             'help' => 'Cleanup done by daily.sh'
         ],
+        'discovery_modules' => [
+            'arp-table' => [
+                'description' => 'arp-table'
```

Comment:
All these descriptions could be a bit better, `ARP Table` for example

Reply:
done